XIV on Mac/Launcher/OTP.swift

```diff
@@ -132,6 +132,8 @@ extension LaunchController {
             msg.accessoryView = txt
             if msg.runModal() == .alertFirstButtonReturn {
                 OTP.store(username: username, secret: txt.stringValue)
+            } else {
+                otpCheck.state = .off
```

Comment:
This was left on intentionally, as the OTP implementation also has an XL like Webserver which is also controlled by that same checkbox

Reply:
The more you know. Reverted.